docker/Dockerfile

```diff
@@ -9,17 +9,27 @@ ENV APP_ROLE=${APP_ROLE}
 ENV DATACENTER=${DATACENTER}
 ENV STARTUP_DELAY=${STARTUP_DELAY}
 ENV EMODB_VERSION=${version}
+ENV CASSANDRA_VERSION=2.1.22
 
 RUN addgroup -S emodb -g 1000
 RUN adduser -S emodb -G emodb -u 1000
 
 COPY --chown=1000 maven/emodb-web-* /app/emodb-web.jar
 
+RUN apk update \
+ && apk add busybox-extras
+
 WORKDIR /app
 
 USER 1000
 
+RUN wget -q http://archive.apache.org/dist/cassandra/${CASSANDRA_VERSION}/apache-cassandra-${CASSANDRA_VERSION}-bin.tar.gz \
```

Comment:
what's the purpose of having C* in emodb web image?

Reply:
To have possibility run nodetool outside of C*

Reply:
could you point to the place where it's used?